CHANGES/5679.misc

```diff
@@ -0,0 +1,5 @@
+Rename endpoints for content to be in plural form
```

Comment:
We are changing in a backward incompatible way our REST API here, so I suggest to add .removal changelog entry and list endpoints which are getting removed/added.
/pulp/api/v3/content/rpm/modulemd/ -> /pulp/api/v3/content/rpm/modulemds/
...

Just to make sure that we inform users about breaking changes. Misc entries are not visible. 
What do you think?

Reply:
Good idea, will make the change.